README.adoc

```diff
@@ -0,0 +1,5 @@
+# Eclipse MicroProfile
+
+This repository is no longer the active process for submitting feature ideas for Eclipse MicroProfile.
+
```

Comment:
Do we need to keep them somewhere?

Each approved proposal will have its own repository created for it that can act as the "source of truth".

Any proposal that isn't "approved", would either be present in the sandbox repository in some intermediate state, or wouldn't exist as no one has created a thread to discuss it.

If we need some central place, then I'd suggest that the Wiki have a link to the sandbox and all repos for each proposal. I wouldn't go beyond that